ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/driver-demoable.json

```diff
@@ -28,6 +28,10 @@
     ],
     "phaseInfo": {
         "phase": "train",
+        "service_endpoint": null,
+        "profile_name": "DEFAULT",
+        "project_id": "<project_id>",
```

Comment:
anomalyDetectionProjectId and anomalyDetectionCompartmentId

Reply:
As discussed offline, moved these to a different block `serviceApiConfiguration.anomalyDetection` instead.